python/cudf/cudf/tests/test_text.py

```diff
@@ -787,6 +787,34 @@ def test_edit_distance():
     assert_eq(expected, actual)
 
 
+def test_edit_distance_matrix():
+    # normal
+    sr = cudf.Series(["rounded", "bounded", "bounce", "trounce", "ounce"])
```

Comment:
just wondering does  this implementation also work for series containing lists of strings?

Reply:
It doesn't. It only accept string type series.